### PR TITLE
Cleanup & fix for ItemRefTooltip

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: KeystoneTooltip.lua [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
+		if ... == '138019' then -- mythic keystone
+			modifierOffset = 16
+		else
+			return
+		end
+	elseif not linkType:find('keystone') then
+		return
+	end
+
+	if mythicLevel and mythicLevel ~= "" then
```
Comment (WetU) [inline KeystoneTooltip.lua]: Parsing mythicLevel for itemLink.

File: KeystoneTooltip.lua [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
+		_, _, _, _, _, _, _, _, _, _, _, _, _, instanceID, mythicLevel = ...
+		if ... == '138019' then -- mythic keystone
+			modifierOffset = 16
+		else
+			return
+		end
+	elseif not linkType:find('keystone') then
+		return
+	end
+
```
Comment (ghost) [inline KeystoneTooltip.lua:15]: Reorderd for mythicLevel from itemLink.

File: KeystoneTooltip.lua [hunk 3]
```diff
@@ -36,32 +35,33 @@ local function GetModifiers(linkType, ...)
...
-			end
-			self:Show()
-		end
-	end
-end
-
-MythicHelperKeystoneTooltip = {}
-function MythicHelperKeystoneTooltip:Init()
-	hooksecurefunc(ItemRefTooltip, 'SetHyperlink', DecorateTooltip) 
-	ItemRefTooltip:HookScript('OnTooltipSetItem', DecorateTooltip)
```
Comment (ghost) [inline KeystoneTooltip.lua:64]: Affix info displayed twice.

**Comment by LeoLeal** [comment]
Fix implemented in another commit.

**Pull request closed without merging**